roles/test_operator/tasks/tempest-tests.yml

```diff
@@ -32,13 +32,20 @@
       register:
         list_allowed
 
+    - name: Prepare includeList for Tempest CR
+      set_fact:
+        allowed_tests: |
+          {% for item in list_allowed.allowed_tests %}
+          {{ item }}
+          {% endfor %}
+
     - name: Set variable
       ansible.builtin.set_fact:
         test_operator_cr: >-
           {{
               test_operator_cr |
               combine({'spec': {'tempestRun': { 'includeList':
-                      list_allowed.allowed_tests | join('\n')
```

Comment:
Isn't this entire change the same as the simple removal of `| join('\n')` or, at most, change it to `| list` ?

Reply:
there is but it doesn't work in the form as is and as i didn't create that one, i had to create a new PR

Reply:
I think that `| list` won't work cause the end result should not be a list, we want:
`
includeList: |
  test1
  test2
`
and not
`
includeList:
  - test1
  - test2
`